usage-guides/api-keys.md

```diff
@@ -0,0 +1,192 @@
+# API Keys
+
+In this section, we will explain various API Keys you can add to rotki.
+Go to `API Keys` menu in the sidebar.
+
+## rotki Premium
+
+You can input the API Key and API Secret needed for rotki premium.
+If you already subscribe, you will see the API key [here](https://rotki.com/home/subscription)
+
+![Set up rotki premium API key/secret pair in an existing account](/images/rotki_premium_set.png)
+
+To replace or disassociate keys with your current account after setting up premium, you can delete it in the same page.
+
+![Delete rotki premium API key/secret pair in a premium account](/images/rotki_premium_set_sync_data.png)
+
+## Exchanges API Keys
+
+![List of connected exchanges](/images/rotki_add_exchange_2.png)
+
+You can integrate many different exchanges with rotki via API Keys.
+Currently supported exchanges are:
+
+- Kraken
+- Poloniex
+- Binance
+- Bitmex
+- Coinbase
+- Coinbase Pro
+- Gemini
+- Bitstamp
+- Binance US
+- Bitfinex
+- Bitcoin.de
+- Iconomi
+- KuCoin
+- Independent Reserve
+- Bitpanda
+- OKX
+- Woo
+- Bybit
+- HTX
+
+![Add API keys for a new exchange](/images/rotki_add_exchange_1.png)
+
+### Steps to Add Exchange API Key:
+
+1. Create API key on your exchange (see [API key permissions](#api-key-permissions))
+2. Navigate to `API Keys > Exchanges` in the left sidebar
+3. Click button `Add an exchange` to open addition menu
+4. Enter your:
+   - API Key
+   - API Secret
+5. Click `Save`
+
+### After Adding:
+
+- Successful addition: Exchange will appear in your list
+- If failed: Verify key and secret are correct
```

Comment:
also point here to the issues that some like binance have due to timestamp difference between the computer and the server. We have info about that